engine/sources/gitdns/source.go

```diff
@@ -247,10 +248,21 @@ func (gs *gitSourceHandler) getAuthToken(ctx context.Context, g session.Group) e
 				}
 				return err
 			}
-			if s.token {
-				dt = []byte("basic " + base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf("x-access-token:%s", dt))))
+
+			if strings.Contains(gs.src.Remote, "bitbucket.org") {
```

Comment:
suggestion: maybe we should just check the host here? e.g. we don't want to do this for something weird like `git@github.com:jedevc/bitbucket.org`

Reply:
thanks 🙏